lib/node_modules/@stdlib/math/base/special/boxcox/examples/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var incrspace = require( '@stdlib/math/utils/incrspace' );
+var boxcox = require( '@stdlib/math/base/special/boxcox' );
```

Comment:
```suggestion
var boxcox = require( './../lib' );
```

This isn't documented anywhere (yet), but project convention is to use the pkg path in the README example and the relative path in the example file.